dev_tools/utils/nxdl_utils.py

```diff
@@ -588,7 +549,7 @@ def get_doc(node, ntype, nxhtml, nxpath):
     if index:
         enum_str = (
             "\n "
-            + ("Possible values:" if len(enums.split(",")) > 1 else "Obligatory value:")
+            + ("Possible values:" if enums.count(",") else "Obligatory value:")
```

Comment:
be explicit, yes if count evals to 0 0 is False but LHS is easier to read to my taste
also !!!negate is !negate but why obfuscate?

Reply:
ok